content: new blog post (#174)
jsheroes/website: src/content/blog/why-speak-at-jsheroes.md

---
author: alex-moldovan
title: Why speak at JSHeroes
published: Nov 8, 2024
tags: ["cfp"]
summary: A few reasons to convince you to apply as a speaker
---

> In 2025 we’ll be running JSHeroes for the 7th time. In all our past editions we had amazing speakers on our stage. Maybe some of you reading this are already convinced and you want to be a speaker at our event, but for everyone else, let me try to articulate the "why".

Reviewer: ```suggestion
In 2025 we’ll be running JSHeroes for the 7th time. In all our past editions we had amazing speakers on stage. Some of you reading this might be already convinced to be a speaker at our event, but for everyone else, let me try to articulate the "why".
```

## Recognition, exposure and benefits

> Our conference has grown steadily and has a very good reputation among tech events. As we have a single track with generous breaks, you will be part of a rather small group of speakers that take the stage every year. Your talk will be seen live by 500+ attendees and then it will be available on our youtube channel to be shared with the entire world.

Reviewer: ```suggestion
Our conference has grown steadily and has a very good reputation among tech events. We have a single track with generous breaks. You will be part of a rather small group of speakers that take the stage every year. Your talk will be seen live by 500+ attendees and then it will be available on our YouTube channel for the entire world.
```

> But we know recognition and exposure don't pay the bills, so you will find that we offer one of the most complete speaker packages out there.

Reviewer: ```suggestion
But we know recognition and exposure don't pay the bills, therefore we offer one of the most complete speaker packages out there.
```

> First of all, we pay for your flights and book for you up to 4 nights at the 5 star hotel where the event takes place. We also redistribute the small profit at the end of the event (usually around 10K EUR - link to transparency) in the form of speaker fees. We’re very proud to have been able to pay 500 EUR to our speakers for the past two editions.

Reviewer: ```suggestion
First of all, we pay for your flights and book for you up to 4 nights at the 5 star hotel where the event takes place. We redistribute the small profit at the end of the event (usually around 10K EUR - link to transparency) in the form of speaker fees. We’re very proud to have been able to pay 500 EUR to our speakers for the past two editions.
```

> On top of that, we arrange for airport transfers to and from the hotel, we offer SIM cards upon arrival and we invite you to a variety of activities with the community and with the organizing team throughout the week.

Reviewer: ```suggestion
On top of that, we arrange for airport transfers to and from the hotel, offer SIM cards upon arrival and invite you to a variety of activities with the community and events with the organizing team throughout the week.
```

## The speaker experience

> If there’s one thing we “obsess” over, it is the speaker experience. We understood from the early days that speakers coming in from all over the world are the best advocates for our event in the future.
> 
> The speaker experience goes beyond the benefits. It encompasses things like having a great MC to prepare the audience for your talk, or making sure your talk is scheduled at the right time of the day so you can bring your best self up on stage. We offer help and support throughout the months before the event, in case you need to rehearse your talk or just brainstorm new ideas.

Reviewer: ```suggestion
We understood from the early days that speakers coming in from all over the world are the best advocates for our event in the future, so the speaker experience is something we are obsessed with improving every year. 

It goes beyond the benefits. It encompasses things like having a great MC to prepare the audience for your talk, or making sure your talk is scheduled at the right time of the day so you can bring your best self up on stage.

We offer help and support throughout the months before the event, in case you need to rehearse your talk or just brainstorm new ideas.
```

> Also, we don’t want you to feel awkwardly put on a pedestal as the superstar of the event. We know most of you dislike that kind of treatment. Maybe you are used to hanging around during the talks in a separate room, the so-called "speakers room" - aka the VIP restricted area. Some events even offer food and drinks there so you never leave that room. We don’t do that. As one of our past speakers nicely framed it: “You finish your talk and then you’re out there in the audience with everyone else”.

Reviewer: ```suggestion
Most of us behind the content of the event did or currently do public speaking, and we know getting  superstar vibes within a conference can feel awkward.

Maybe you are used to hanging around during the talks in a separate room, the so-called "speakers room"&mdash;aka the VIP restricted area. Some events even offer food and drinks there so you never leave that room. We don’t do that. As one of our past speakers nicely framed it: “You finish your talk and then you’re out there in the audience with everyone else”. (I would link the YouTube video with the timestamp here)
```

> Of course, the hotel is a big advantage here here. You always have your room if you need some focus/alone time. We found that this arrangement significantly improves the experience for speakers, as well as for the audience, as it facilitates networking and meaningful conversations during the breaks.

Reviewer: ```suggestion
Of course, the hotel is a big advantage here. You always have your room if you need some rest from socializing. We found this arrangement significantly improves the experience for speakers, as well as for the audience. It facilitates networking and meaningful conversations during the breaks with the opportunity to take a break if you need to.
```

> Finally, most of you will be coming to Romania for the first time. So we want part of the experience to be an introduction into our culture and places. The day before the event, we organize a tour of the old town with a local professional guide. We choose dinner venues where you can sample local food. And if the weather permits it, we hang out together for a full day trip on the Saturday after the conference. We did hikes, picnics and barbeques in the past, and you can see the vibes around that in the images below. This is personally my favorite day of the whole event.

Reviewer: ```suggestion
Finally, most of you will be coming to Romania for the first time. So we want part of the experience to be an introduction into our culture and places. The day before the event, we organize a tour of the old town with a local professional guide.

We choose dinner venues where you can taste local food, and if the weather permits it, we hang out together for a full day trip on the Saturday after the conference.

We did hikes, picnics and barbeques in the past, and you can feel the vibes around those in the images below. This is personally my favorite day of the whole event.
```

WIP Pics

## Call for papers

WIP section

> If the arguments here convinced you that JSHeroes is the right conference for your talk, we’d love to see you submit a proposal until the end of December.

Reviewer: ```suggestion
If the arguments here convinced you JSHeroes is the right conference for your talk, we’d love to see you [submit a proposal](TODO: cfp link) until the end of December.
```
